### PR TITLE
docs: Post install doc update

### PR DESCRIPTION
**Pull request by atmandhol**
# Which other branches do you want a technical writer to cherry-pick this PR to (if any)?

It's best to PR to the most recent relevant branch and leave all the cherry-picking to the
writer, except where earlier branches require factual changes to the content.
For more information about the branches, see https://github.com/pivotal/docs-tap#branches

### CODE REVIEW

**Comment by atmandhol** [comment]
@kcoriordan please review and merge!

File: supply-chain/platform-engineering/how-to/installing-supply-chain/post-install-configuration.hbs.md [hunk 1]
```diff
@@ -4,80 +4,86 @@
...
 
-## Namespace Provisioner
+## Things to configure
 
-The recommended way of configuring service accounts and permissions on Tanzu Application Platform is
-to use the Namespace Provisioner.
+- OCI Store Configuration
+  - Supply Chains persist data between stages by reading and writing to an OCI repository.  The location of the OCI repository is configured by a Kubernetes Secret named `oci-store` that exists within the developer namespace.
+  - Access to this repository is controlled by a tekton annotated secret which can have any name as long as it has the `tekton.dev/docker-0` annotation pointing to the OCI repository.
+- Permissions for Buildpacks Cluster Builders for buildpack-build component
```
Comment (kcoriordan) [inline supply-chain/platform-engineering/how-to/installing-supply-chain/post-install-configuration.hbs.md:15]: I presume TBS is Tanzu Build Service?

File: supply-chain/platform-engineering/how-to/installing-supply-chain/post-install-configuration.hbs.md [hunk 2]
```diff
@@ -4,80 +4,86 @@
...
+  namespace: tap-install
+type: Opaque
+stringData:
+  ocistore.yaml: |
+    tanzusupplychain:
+      ocistore:
+        username: <registry-username>
+        password: <registry-password>
+        server: <registry-server>
+        repository: <registry-repo>
```
Comment (kcoriordan) [inline supply-chain/platform-engineering/how-to/installing-supply-chain/post-install-configuration.hbs.md:39]: Can you call out the expected value for these variables, and I will add them in.

File: supply-chain/platform-engineering/how-to/installing-supply-chain/post-install-configuration.hbs.md [hunk 3]
```diff
@@ -4,80 +4,86 @@
...
     supply_chain_service_account:
       secrets:
-        # List your git & docker secrets here
-        - git-secret
-        - my-oci-store-credentials # this secret is described below
+      - oci-store-credentials
 ```
 
-## OCI Store
+## Create Developer Namespaces
```
Comment (kcoriordan) [inline supply-chain/platform-engineering/how-to/installing-supply-chain/post-install-configuration.hbs.md:62]: Is create Developer Namespaces a consecutive step after  the previous step to - configure Namespace Provisioner to use the accelerator sample?
Or is it maybe a prereq step for some other procedure?

**Pull request merged**